docs/DEVELOPMENT.md

```diff
@@ -12,82 +12,109 @@ install -m 755 operator-sdk_linux_amd64 ${SOME_DIR_IN_YOUR_PATH}/operator-sdk
+- You will login and see _view token_.  Click on that to see
+- **Your API token is**.  The next line is your full token
+- _podman login_ with your username and the token as your password should put the token in your AUTH_FILE
+
+#### Using the token
+If docker build does not automatically use the token in the default authfile, it can be forces:
+```shell
+export AUTH_FILE='~/.docker/config.json'
+```
+
 In summary:
 - login to one of the clusters' console
 - use the console's shortcut to get the commandline login command
 - log in from the command line with the provided command
-- use "oc registry login" to save the token locally
+- use "podman login" to save the token locally
+- set AUTH_FILE to force docker-build to use the token from your file if needed
 
-### Using public images
-
-If you cannot login to registry.ci.openshift.org, a temporary solution is to use
-public images during build and test. At the time of writing, the following public images
-does the trick.
-
-```shell
-export BUILDER_IMAGE=registry.ci.openshift.org/openshift/release:golang-1.21
-export TARGET_IMAGE=registry.ci.openshift.org/origin/4.16:base
-make docker-build
-```
-
-## Set Environment Variables
+## Change catalog for private quay registry
+### Set Environment Variables
 
 Set your quay.io userid
-```
+```shell
 export QUAY_USERID=<user>
-```
-
-```
 export IMAGE_TAG_BASE=quay.io/${QUAY_USERID}/openshift-sandboxed-containers-operator
-export IMG=quay.io/${QUAY_USERID}/openshift-sandboxed-containers-operator
 ```
 
 ## Viewing available Make targets
 ```
 make help
 ```
-
 ## Building Operator image
 ```
 make docker-build
 make docker-push
 ```
+### Skipping tests in Make
+Add SKIP_TESTS=1 when calling make.  ex:
+```
+make SKIP_TESTS=1 docker-build
+```
 
 ## Building Operator bundle image
 
-If you are deploying in an OpenShift cluster then modify the
-value of the env variable `SANDBOXED_CONTAINERS_EXTENSION` to `sandboxed-containers`
-in the file `config/manager/manager.yaml` before running the below mentioned
-commands.
+### If you are deploying in an OpenShift cluster
+Modify SANDBOXED_CONTAINERS_EXTENSION in config/manager/manager.yaml before you __Make the bundle__
+```shell
+sed -ie 's/kata-containers/sandboxed-containers/' config/manager/manager.yaml
+```
 
+### Make the bundle
 ```
 make bundle CHANNELS=candidate
 make bundle-build
 make bundle-push
 ```
 
-
 ## Building Catalog image
+#### Overwrite/create catalog/index.yaml if needed
+If `IMAGE_TAG_BASE` was set for your private registry, `catalog/index.yaml` will need to be rewritten.  Before running `make catalog-build`, run this:
+
+```
+DEFAULT_CHANNEL="stable"  # from Makefile
+VERSION=1.7.0             # from Makefile
+BUNDLE_IMG=${IMAGE_TAG_BASE}-bundle:v${VERSION}
+make opm                  # to get bin/opm
+rm -rf catalog && mkdir -p catalog
+bin/opm init  sandboxed-containers-operator --default-channel=${DEFAULT_CHANNEL} --description ./README.md --output yaml > catalog/index.yaml
```

Comment:
I know this was discussed previously at https://github.com/openshift/sandboxed-containers-operator/pull/422#discussion_r1661202637 but something now looks wrong to me.

This `bin/opm init ...` has a dependency on `README.md` : it should be run again every time `README.md` is changed, not just once. Typical case for a rule in the Makefile.

Also I still don't understand why `catalog/index.yaml` should be checked in the repo.

@cpmeadors ?